git-quick-stats

```diff
@@ -202,19 +204,20 @@ showMenu() {
     printf %b "${NUMS}    3)${TEXT} Git changelogs (last $_limit days)\\n"
     printf %b "${NUMS}    4)${TEXT} Git changelogs by author\\n"
     printf %b "${NUMS}    5)${TEXT} My daily status\\n"
-    printf %b "${NUMS}    6)${TEXT} Save git log output in JSON format\\n"
+    printf %b "${NUMS}    6)${TEXT} Contribution stats (by author) on a specific branch -> output is a csv\\n"
```

Comment:
Let's change `Contribution stats (by author) on a specific branch -> output is a csv` to `Output daily stats by branch in CSV format`